std/file.d

```diff
@@ -215,18 +215,6 @@ Read entire contents of file $(D name) and returns it as an untyped
 array. If the file size is larger than $(D upTo), only $(D upTo)
 bytes are read.
 
-Example:
-
-----
-import std.file, std.stdio;
-void main()
-{
-   auto bytes = cast(ubyte[]) read("filename", 5);
```

Comment:
Where is DDoc-ed version? Removing it just because it can't be tested as is not good.